doc/man/8/rbd.rst

```diff
@@ -543,8 +543,10 @@ Commands
 
 :command:`mirror pool info` [*pool-name*]
   Show information about the pool or namespace mirroring configuration.
-  For a pool, it includes mirroring mode, peer UUID, remote cluster name,
-  and remote client name. For a namespace, it includes only mirroring mode.
+  For a pool, it includes mirroring mode,site name, mirror uuid, remote
+  namespace, peer UUID, remote cluster name, and remote client name. For a
+  namespace, it includes only the mirroring mode, the mirror uuid and remote
```

Comment:
UUID (uppercase twice)

Reply:
I see `uuid` used elsewhere in the file. Should I just change `UUID` to `uuid` here?

Reply:
I support Anthony's suggestion to keep/switch to uppercase.  I see only one instance of lowercase spelling in actual English text as opposed to `:command:` stanzas.

Reply:
Ok. I couldn't quite understand Anthony's original comment - it was not clear whether it was supposed to be capitalized or not.